package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
why is package.json needed?

Can we just use submodules like:

https://github.com/yieldnest/yieldnest-protocol

That is strongly preferred, we'd like not to use this at all if we can just submodules.

Reply:
@danoctavian Unfortunately it hasn't been possible to install all the dependencies through foundry. There's some discrepancy in the different dependencies on the LayerZero side. 

Particularly the upgradeable version at https://github.com/JorgeAtPaladin/LayerZero-v2/tree/lz-upgrade

They use a combination of both yarn and forge (gitmodules) to manage the dependencies. I thought it is cleaner to use only one of them. But if you would prefer I can move as many of the dependencies as possible to .gitmodules and use whatever is necessary using npm (or yarn). Unfortunately the way LazerZero is publishing their contracts they seem be best installed from npm.

Reply:
Noted thanks @dan13ram  that is understood

If we can't eliminate it completely, this is fine.

At least we made it a separate repo, makes it easier to  deal with it in isolation